ConanInstall.cmake

```diff
@@ -59,28 +59,17 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
     ${CONAN_QT}
     openssl/1.1.0l # ruby 2.5.5 won't work with 1.1.1x, so use 1.1.0l here to try to force every package to align on the same as ruby
     # Track NREL/stable in general, on a feature branch this could be temporarily switched to NREL/testing
-    openstudio_ruby/2.5.5@nrel/stable    # TODO: Temp #@nrel/stable
-    boost/1.71.0
-    #boost_asio/${BOOST_VERSION}@bincrafters/stable
-    #boost_program_options/${BOOST_VERSION}@bincrafters/stable
-    #boost_regex/${BOOST_VERSION}@bincrafters/stable
-    #boost_filesystem/${BOOST_VERSION}@bincrafters/stable
-    #boost_crc/${BOOST_VERSION}@bincrafters/stable
-    #boost_algorithm/${BOOST_VERSION}@bincrafters/stable
-    #boost_uuid/${BOOST_VERSION}@bincrafters/stable
-    #boost_log/${BOOST_VERSION}@bincrafters/stable
-    #boost_numeric_ublas/${BOOST_VERSION}@bincrafters/stable
-    #boost_functional/${BOOST_VERSION}@bincrafters/stable
-    #boost_geometry/${BOOST_VERSION}@bincrafters/stable
-    pugixml/1.10@bincrafters/stable
-    jsoncpp/1.9.2
-    zlib/1.2.11@nrel/testing # TODO: Temp, pending merging of https://github.com/conan-io/conan-center-index/pull/1526, to resolve #3961
-    fmt/6.2.0
-    sqlite3/3.30.1
-    cpprestsdk/2.10.14@bincrafters/stable
-    websocketpp/0.8.1@bincrafters/stable
-    geographiclib/1.49@bincrafters/stable
-    swig_installer/4.0.1@bincrafters/stable
+    openstudio_ruby/2.5.5@nrel/testing    # TODO: Temp #@nrel/stable
```

Comment:
Recipe done with this PR: https://github.com/NREL/conan-openstudio-ruby/pull/28

Reply:
Are just waiting for that conan recipe to pass CI and push upstream? And then once that conan package is available we can change openstudio_ruby/2.5.5@nrel/testing to openstudio_ruby/2.5.5@nrel/stable?

Reply:
Yes. I need a mac to debug why conan openstudio ruby is unhappy on the travis mac, but I failed to connect the NREL VPN yesterday after repeated attempts (not sure whether I forgot my password or my access is blocked). I need to pull my old MBP